ocaml/default.nix

```diff
@@ -1470,8 +1471,6 @@ with oself;
         [ ./core_unix.patch ] else [ ];
 
     postPatch = ''
-      ${o.postPatch}
```

Comment:
if you don't have `postPatch` it means that you are not running the patched 0.15 sources. Fix that instead of deleting this.

Reply:
Why did that come in here? :thinking: